src/status_im2/contexts/quo_preview/wallet/wallet_activity.cljs

```diff
@@ -0,0 +1,218 @@
+    :value "Network: Multinetwork"}
+   {:key   moonpay
+    :value "Market: Moonpay"}
+   {:key   binance
+    :value "Market: Binance"}])
+
+(def prefixes
+  [{:key   :t/to
+    :value "To"}
+   {:key   :t/in
+    :value "In"}
+   {:key   :t/via
+    :value "Via"}
+   {:key   :t/from
+    :value "From"}
+   {:key   :t/on
+    :value "On"}
+   {:key   :t/at
+    :value "At"}])
+
+(def descriptor
+
+  (concat
+   [{:key     :blur?
+     :type    :select
+     :options [{:key true}
+               {:key false}]}
+    {:key     :transaction
+     :type    :select
+     :options [{:key :receive}
+               {:key :send}
+               {:key :swap}
+               {:key :bridge}
+               {:key :buy}
+               {:key :destroy}
+               {:key :mint}]}
+    {:key     :status
+     :type    :select
+     :options [{:key :pending}
+               {:key :confirmed}
+               {:key :finalised}
+               {:key :failed}]}
+    {:key  :timestamp
+     :type :text}
+    {:key     :counter
+     :type    :select
+     :options [{:key   1
+                :value 1}
+               {:key   2
+                :value 2}
+               {:key   3
+                :value 3}]}
+
+    {:label   "Slot 1"
+     :key     :first
+     :type    :select
+     :options context-tags}
+    {:label   "Slot 2 prefix"
+     :key     :second-prefix
+     :type    :select
+     :options prefixes}
+    {:label   "Slot 2"
+     :key     :second
+     :type    :select
+     :options context-tags}
+    {:label   "Slot 3 prefix"
+     :key     :third-prefix
+     :type    :select
+     :options prefixes}
+    {:label   "Slot 3"
+     :key     :third
+     :type    :select
+     :options context-tags}
+    {:label   "Slot 4 prefix"
+     :key     :fourth-prefix
+     :type    :select
+     :options prefixes}
+    {:label   "Slot 4"
+     :key     :fourth
+     :type    :select
+     :options context-tags}]))
+
+(defn view
+  []
+  (let [component-state (reagent/atom {:transaction   :send
+                                       :timestamp     "Today 22:20"
+                                       :status        :pending
+                                       :counter       1
+                                       :first         asset-snt
+                                       :second-prefix :t/from
+                                       :second        piggy-bank
+                                       :third-prefix  :t/to
+                                       :third         aretha-gosling
+                                       :fourth-prefix :t/via
+                                       :fourth        mainnet
+                                       :blur?         false})]
+    (fn []
+      [preview/preview-container
+       {:state                 component-state
+        :descriptor            descriptor
+        :blur?                 (:blur? @component-state)
+        :show-blur-background? true}
+       [rn/view {:style {:align-self :center}}
```

Comment:
FYI: `preview-container` accepts a `component-container-style` to avoid the common need to wrap the children in a `view`. I've seen a handful of previews that could take advantage of this prop.

Maybe `component-container-style` is too long of a name, I guess just `container-style` works well. I'll try to remember to rename this if nobody does before me.